service/src/utils/onchain-data.ts

```diff
@@ -10,18 +13,73 @@ export interface OnchainProfile {
     rewards: BigNumber;
 }
 
+export interface SarcophagusData {
+    id: string,
+    resurrectionTime: Date
+}
+
 interface InMemoryStore {
-    sarcophagi?: string[],
+    sarcophagi: SarcophagusData[],
+    unwrappedSarcophagi: string[],
```

Comment:
does unwrappedSarcophagi need to be tracked? if a sarcophagi is unwrapped, it could be removed from the `sarcophagi` array, rather than being pushed to a different array.

Reply:
This was me thinking about how to potentially deal with the case where the archaeologist has unwrapped a sarcophagus that others failed to unwrap in time but that hasn't yet been cleaned.

Very initial thoughts, I dunno. It might be removed or changed as we think through these edge cases anyway.

Reply:
I was thinking the `getOnchainCursedSarcophagi` function would have some conditional logic to determine if a sarcophagus has been unwrapped by the arch but not yet cleaned, and if so, wouldn't be pushed to the `sarcophagi` array. 

Also, if we *do* push it to the `sarcophagi` array, that means we also need to make sure an unwrap job has been scheduled for that sarcophagi. Since the state is being rebuilt on an interval, there is a good chance that job already exists so don't need to schedule it.